_projects/openindexmaps/index.markdown

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
Can/should we separate this change into a separate pull request? Seems like something we could move forward with?